src/components/AstronautsPage/components/AstronautsList/AstronautsList.tsx

```diff
@@ -1,4 +1,26 @@
 import React from 'react';
+import { Grid } from '@mui/material';
 import { AstronautsItem } from './components/AstronautsItem';
+import { useAppSelector } from '../../../../store';
+import type { Astronauts } from '../../types/Astronauts';
 
-export const AstronautsList = () => <AstronautsItem />;
+export const AstronautsList = () => {
+  const astronauts = useAppSelector(state => state.astronauts.astronauts);
```

Comment:
it is not app selector

Reply:
changed on useSelector with Astronauts selector